go.mod

```diff
@@ -215,3 +215,5 @@ require (
 )
 
 require github.com/samber/lo v1.33.0
+
+require gotest.tools/v3 v3.3.0 // indirect
```

Comment:
where did that come from?

Reply:
go mod tidy just added this automatically for some reason

Reply:
if you remove the `go.mod`/`go.sum` changes and run go mod tidy again, will it do the same?